parsec-mock.py

```diff
@@ -1,2 +1,94 @@
 # Copyright 2021 Contributors to the Parsec project.
 # SPDX-License-Identifier: Apache-2.0
+import click
+
+import socket
+import os
+from os import listdir
+from os.path import isfile, join
+
+import base64
+
+from yaml import safe_load
+
+
+@click.command()
+@click.option('--test-folder', default='./testdata', help='Load all tests from folder')
+@click.option('--parsec-socket', default='./parsec.sock', help='Path to parsec unix socket')
+def run_test(test_folder, parsec_socket):
+    print('Mock parsec service listening on unix://{}.'.format(parsec_socket))
```

Comment:
Shouldn't it be `unix:{}`? Though I have to admit, at this point I'm not really sure what the correct format for that scheme is